src/bp/ui-admin/package.json

```diff
@@ -48,6 +51,7 @@
     "eject": "react-scripts eject"
   },
   "devDependencies": {
+    "@blueprintjs/core": "^3.15.1",
     "cross-env": "^5.2.0",
```

Comment:
types/react & types/react-dom should be dev-dependencies
blueprintjs/core should be dependencies
react-tooltip should be removed, use  blueprintjs Tooltip